src/workspace-data/data-table/shared/DataTable.js

```diff
@@ -634,6 +634,7 @@ const DataTable = (props) => {
                       const editLink =
                         editable &&
                         dataProvider.features.supportsEntityUpdating &&
+                        !updatingEntity &&
```

Comment:
This is a secondary fix - the "Edit value" tooltip associated with the edit icon could remain visible even when the icon itself was gone.